src/App.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | undefined,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: undefined,
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.handleDocumentClick);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.handleDocumentClick);
+  }
+
+  handleDocumentClick = (event: KeyboardEvent) => {
+    this.setState({
```

Comment:
It's okay but naming could be better. Let's name the function in a depend of what it do. It's not a click handler but keyup handler. So `handleDocumentKey` would be better.